### PR TITLE
Adding a trailing slash when moving a file to a directory

### PR DESCRIPTION
**Pull request by MauJFernandezUVR**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -76,9 +76,11 @@ cp -a myMusic myMedia
 **STOP DRAG AND DROP A FILE, OR CTRL + X, CTRL + V A FILE**
 
 ```
-mv readme.txt documents
+mv readme.txt documents/
 ```
 
+[For this reason](http://unix.stackexchange.com/a/50533)
```
Comment (stevemao) [inline readme.md]: Maybe reword this to 

Trailing slash for this reason?

Comment (MauJFernandezUVR) [inline readme.md]: Yeah you're right, I did the PR fast and didn't notice that, thanks.
I think instead of that, I'd rather put the warning there because it's a common error that can cause a huge mess, what do you thinks of this?:

**Always** use a trailing slash when moving files, [for this reason](http://unix.stackexchange.com/a/50533)

Comment (cht8687) [inline readme.md]: @MauJFernandezUVR  sounds good!

Comment (MauJFernandezUVR) [inline readme.md]: Awesome, I just pushed the new commit.

**Comment by cht8687** [comment]
it's quite clear now. 
Glad to learn this tip! :+1:

**Comment by stevemao** [comment]
@MauJFernandezUVR awesome work!!

**Comment by MauJFernandezUVR** [comment]
Thanks! Glad to help.
